compliance/compliance-server/migrations/2024-06-28-231756_create_compliance_table/up.sql

```diff
@@ -2,6 +2,7 @@
 CREATE TABLE IF NOT EXISTS wallet_compliance (
     address TEXT PRIMARY KEY,
     is_compliant BOOLEAN NOT NULL,
+    risk_level TEXT NOT NULL,
```

Comment:
is it safe to edit migrations like this 😅 or should we just make a new one? though I get that nothing is in prod yet so its chill

Reply:
no it is not safe, as you mention this is only done because nothign is deployed